### PR TITLE
Export command (incl. filters, cli, and tests)

### PR DESCRIPTION
**Pull request by eode**
This is the 'export' command, with future items (according to the spec doc) excluded.

There are some minor changes (some arg names, arg position for output dir) but all of the functionality is there.

It should be a short path to some of the desired "future" items in the specification as well, when our focus shifts there.

### CODE REVIEW

**Comment by akarve** [comment]
@asah @eode is it possible to catch `quiltdata:tensorflow_virtual_files` up with `quiltdata:master` so CI doesn't keep failing?

**Comment by quiltdata** [comment]
already done.

ᐧ

File: compiler/quilt/test/test_command.py [hunk 1]
```diff
@@ -17,6 +17,10 @@
 from quilt.tools import command, store
 from .utils import QuiltTestCase, patch
 
+from ..tools.compat import pathlib
+from ..tools.compat import tempfile
```
Comment (asah) [inline compiler/quilt/test/test_command.py]: one line pls - import pathlib, tempfile

File: compiler/quilt/tools/command.py [hunk 2]
```diff
@@ -1234,24 +1234,36 @@ def update(pkginfo, content):
...
+    :param filter: function -- takes a node path list, returns True to export
+    :param filename_mapper: function -- takes and returns a node path list
     """
-    # TODO: filters
-    # TODO: tests
-    # TODO: export via symlinks / hardlinks (unwise / messing with datastore? windows compat?)
-    if filter is not None:
-        raise NotImplemented("filters not implemented yet")
+    # TODO: Update docs
+    # TODO: force
```
Comment (asah) [inline compiler/quilt/tools/command.py]: bad merge?  I added force already (and needed it!)

Comment (eode) [reply compiler/quilt/tools/command.py]: Bad merge maybe.  I'll kill this PR and retrace my steps.

Comment (eode) [reply compiler/quilt/tools/command.py]: At the time it was created, it worked and was based on master, but moving it to tensorflow_virtual_files and the merge between our code there killed it.

**Comment by eode** [comment]
bad merge after making PR.  Closed until sorted.

**Pull request closed without merging**